corehq/apps/app_manager/suite_xml/xml_models.py

```diff
@@ -131,6 +131,11 @@ class Text(XmlObject):
     locale_id = StringField('locale/@id')
 
 
+class Hint(XmlObject):
```

Comment:
I'm surprised we don't have something that does this already... how do we do this in other cases with help text?

Reply:
I'd think `Hint` would be a subclass of `Display`, just with a different ROOT_NAME.

Reply:
@orangejenny can a `Hint` have media attached to it? If so, then I agree with you. `Display` also includes `media_image` and `media_audio`.

Reply:
Yes, we're not adding HQ UI support for media right now but do want the suite to support adding it in the future.

Reply:
> I'd think Hint would be a subclass of Display, just with a different ROOT_NAME.

That means, `Display.hint` would be a `Hint` that extends a `Display` again, which is recursive definition. I am not sure how a recursive element composition could work

Reply:
I don't quite follow, why is that a problem? Elements can be nested inside elements of the same type, `Detail` does this.

Reply:
Ah, nice. I have updated it.